app/models/order.rb

```diff
@@ -3,6 +3,16 @@ class Order < ActiveRecord::Base
   has_many :order_staches
   has_many :staches, through: :order_staches
 
+  validates :zipcode, length: { is: 5 }
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zipcode, presence: true
+  validates_inclusion_of :status,
+    in: ['ordered', 'paid', 'completed','cancelled']
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for an array of words.